Framework/Email.php

```diff
@@ -63,30 +63,87 @@ public function sendEmail($from, $fromName, $toAdress, $subject, $content, $sent
         }
     }
 
+    /**
+     * 
+     * Send an Email to all users or all managers within a space
+     *
+     * @param array $params
+     *          required to fill sendEmail() parameters. Depends on why we want to notify space admins
+     * @param string $lang
+     * 
+     * @return string result of call to function sendMail() telling if mail was sent or not
+     */
+    public Function sendEmailToSpaceMembers($params, $lang = "") {
+        $modelSpace = new CoreSpace();
+        $spaceId = $params["id_space"];
+        $from = Configuration::get('smtp_from') /* $params["from"] */;
+        $spaceName = $modelSpace->getSpaceName($spaceId);
+        $subject = $params["subject"];
+        $fromName = "Platform-Manager";
+        $subject = CoreTranslator::MailSubjectPrefix($spaceName, $lang) . " " . $subject;
+
+        // get the emails
+        $toAddress = array(); 
+        switch ($params["to"]) {
+            case "all":
+                $toAddress =
+                    $this->formatAddresses($modelSpace->getEmailsSpaceActiveUsers($spaceId));
+                break;
+            case "managers":
+                $toAddress =
+                    $this->formatAddresses($modelSpace->getEmailsSpaceManagers($spaceId));
+                break;
+            default:
+                // TODO: test that! => linked to booking module. Create a dedicated function ?
+                $modelCalEntry = new BkCalendarEntry();
```

Comment:
nope, Email  should not have anything related to modules but user or space models (core module)

module calling this should in this case get email addresses and call Email with a list of emails